src/components/Grid/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { GridLayout } from './Grid.styled';
+
+const Grid = ({ children, col }) => {
+    return(
```

Comment:
You can remove 'return' here and make it shorter

Reply:
Removed